app/assets/v2/js/base.js

```diff
@@ -16,6 +16,23 @@ $(document).ready(function() {
 
   force_no_www();
 
+  var record_campaign_to_cookie = function() {
+    var paramsStr = decodeURIComponent(window.location.search.substring(1));
+
+    paramsStr.split('&')
+      .map((paramStr) => {
+        var [ key, value ] = paramStr.split('=');
+
+        return {key, value};
+      })
+      .filter((param) => [ 'utm_medium', 'utm_source', 'utm_campaign' ].indexOf(param.key) !== -1)
```

Comment:
Nit - can we use [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) here?

```suggestion
      .filter((param) => [ 'utm_medium', 'utm_source', 'utm_campaign' ].includes(param.key))
```

Reply:
you no need to support IE?

Reply:
Hmm....I can see [babel in the devDependencies](https://github.com/gitcoinco/web/blob/master/package.json#L24)...and it looks like it's used in the [Webpack config](https://github.com/gitcoinco/web/blob/master/webpack.config.js#L35) (but I'm not a Webpack expert, TBH) so I think IE should be supported.

Reply:
Ahhh....fair enough. Suppose babel config could be updated...but likely out of scope for this PR.